### DISCUSSION TITLE
Coordinating Human Rights / Digital Identity / Blockchain Law Advocacy Volunteers

### DISCUSSION THREAD

**Discussion by ChristopherA** [BlockchainCommons/Community#44] [Ideas]
We are seeking volunteers to help us track and advocate for the best possible laws and regulations that support human rights, digital identity, decentralized technologies such as blockchains, responsible key management/custody, etc. especially from our perspective as technologists who support human dignity.

We've had some success in Wyoming, and to a lesser extent in California, and some influence in Europe, Canada & New Zealand.

At first we just want to capture where the action is — what laws and regulations already exist, which are in progress, who is involved, upcoming meeting, which laws and proposals are the best, and if our voice can add value. We are all volunteers on this so we don't want to tilt at windmills!

A place to start is what is to document going on in Wyoming. The Select Committee on Blockchain has a number of meetings coming up this year https://www.wyoleg.gov/Committees/2021/S19 and I'm already on some of the subcommittees, in particular Digital Identity and DAOs. We should create a repo here in GitHub that links to the existing Wyoming Laws, maybe points out to key parts for non-lawyers, and what changes we advocate.

Add your name to this discussion if you are interested in volunteering!

-- Christopher Allen

**Comment by sunintb** [comment]
Interested in helping!

**Comment by abrahamross** [comment]
I would definitely be interested with helping with this! I will be at the select committee in person in September anyways also so I could help prepare and write up reports for that.

**Comment by zi-sakura** [comment]
interested in pitching in.

**Comment by ChristopherA** [comment]
@zi-sakura @abrahamross @zi-sakura Let's schedule a call, maybe Wednesday the 30th, to discuss getting this started in HackMD.

**Reply by abrahamross** [reply]
I could do Wednesday or Thursday of next week around the same time that we have usually been meeting if that works for everyone else. I can be flexible though depending on the day.

**Comment by vashto** [comment]
We've talked about me participating in this topic too in our meetings. Just popping a note in here as well for when we set up the call. Either day is good.

**Comment by ChristopherA** [comment]
Let’s plan for Wednesday at the usual tim of 11am PDT.

**Comment by iamvkv** [comment]
I'm late to this, but am interested in joining the next call.

**Comment by lexvoid** [comment]
Absolutely in, where do I sign up?

**Comment by Ant0n3lla** [comment]
I am interested in volunteering in it about UK,France and Italian laws and regulations

**Comment by ChristopherA** [comment]
@Ant0n3lla There is now a start at https://github.com/BlockchainCommons/law-and-advocacy.  Lots of oddball tasks are needed, including converting a list of laws & some diagrams from HackMD to that repo.
